euth_wagtail/assets/scss/components/_howto.scss

```diff
@@ -0,0 +1,110 @@
+.howto__container {
```

Comment:
i introduced this file, because i think it is a lot of very specific styles for the howto screen.
what do you think @phillimorland ?

Reply:
totally agree!